pyface/ui/qt4/action/tool_bar_manager.py

```diff
@@ -168,6 +186,20 @@ def __init__(self, tool_bar_manager, parent):
 
         return
 
+    def _remove_event_listeners(self):
```

Comment:
Same here...from the name I'd expect only calls to remove the handlers but this method is doing more than that. Should this be named `dispose` as well?